e2e-test/src/tests/analyze.test.js

```diff
@@ -0,0 +1,106 @@
+import { vBtn, vListTile } from 'jest-puppeteer-vuetify';
+import { uploadDataset } from '../util';
+
+
+describe('download dataset', () => {
+  it('transform table', async () => {
+    await expect(page).toClickXPath(vBtn('Your Datasets'));
+
+    await uploadDataset('tableTest');
+    await expect(page).toClickXPath(vListTile({ title: 'tableTest.csv ', content: 'Dataset ready for analysis.' }) + vBtn('View Data'));
+
+    // click transform table
+    await expect(page).toClickXPath(vListTile({ title: 'Transform Table' }));
+
+    // change normalize to 'Min Max'
+    await expect(page).toClickXPath("//label[@class='v-label theme--light'][contains(text(), 'Min Max')]");
+    await expect(page).toClickXPath(vListTile({ title: 'Download Data' }));
+
+    // verify the value has been changed: hard coded but there might be a better way to do that
+    await page.waitForXPath("//div[@class='vue-recycle-scroller__item-view'][1]//div[@class='column']/div[@class='cell type-sample'][1][contains(text(), '0.249')]");
+    await expect(page).toClickXPath(vListTile({ title: 'Transform Table' }));
+
+    // change transform to 'log 10'
+    await expect(page).toClickXPath("//label[@class='v-label theme--light'][contains(text(), 'Log 10')]");
+
+    // click download data to verify the change
+    await expect(page).toClickXPath(vListTile({ title: 'Download Data' }));
+    await page.waitForXPath("//div[@class='vue-recycle-scroller__item-view'][1]//div[@class='column']/div[@class='cell type-sample'][1][contains(text(), '-0.603')]");
+
+    await expect(page).toClickXPath(vListTile({ title: 'Transform Table' }));
+
+    // change scale to 'Pareto Scaling'
+    await expect(page).toClickXPath("//label[@class='v-label theme--light'][contains(text(), 'Pareto Scaling')]");
+    await expect(page).toClickXPath(vListTile({ title: 'Download Data' }));
+    await page.waitForXPath("//div[@class='vue-recycle-scroller__item-view'][1]//div[@class='column']/div[@class='cell type-sample'][1][contains(text(), '0.393')]");
```

Comment:
I don't know how to verify the plot has changed. I could take screenshot but I don't what would I compare that to.
I have no idea how to verify that the graph has changed.